tests/Operation/FindAndModifyFunctionalTest.php

```diff
@@ -4,6 +4,7 @@
 
 use MongoDB\Driver\BulkWrite;
 use MongoDB\Model\BSONDocument;
+use MongoDB\Operation\Explain;
```

Comment:
This may be a leftover from when you were testing explain functionality in various operation tests.